absl/debugging/internal/vdso_support.cc

```diff
@@ -50,6 +50,18 @@
 #define AT_SYSINFO_EHDR 33  // for crosstoolv10
 #endif
 
+#if defined(__FreeBSD__)
+
+#if !defined(Elf64_aux_t)
+using Elf64_aux_t = Elf64_Auxinfo;
```

Comment:
Did you mean `auxv_t` ?

This isn't going to work -- it's not `#define`d. On a Linux system we have:
```
# 1135 "/usr/include/elf.h" 3 4
typedef struct
{
  uint32_t a_type;
  union
    {
      uint32_t a_val;
    } a_un;
} Elf32_auxv_t;

typedef struct
{
  uint64_t a_type;
  union
    {
      uint64_t a_val;
    } a_un;
} Elf64_auxv_t;
```
So I think you need:
```
#if defined(__FreeBSD__)
using Elf32_auxv_t = Elf32_Auxinfo;
using Elf64_auxv_t = Elf64_Auxinfo;
#endif 
```